Firmware/language_en.h

```diff
@@ -101,6 +101,9 @@
 #define(length=20) MSG_CHANGING_FILAMENT	"Changing filament!"
 
 
+#define MSG_FILAMENT_NOTIFICATION_MODE_CONTINUOUS					"Fil Notif  [continuous]"
```

Comment:
The MSG is 23 characters long, the limit is 20 on the display as no chars are before or behind.
I think this would fit better
#define MSG_FILAMENT_NOTIFICATION_MODE_CONTINUOUS					"Fil.Notify  [long]"
#define MSG_FILAMENT_NOTIFICATION_MODE_SHORT					        "Fil.Notify [short]"
#define MSG_FILAMENT_NOTIFICATION_MODE_SILENT					        "Fil.Notify [silent]"